english/modern-tech

```diff
@@ -0,0 +1,2 @@
+If you would like to learn more about today's modern tech please Click [here](http://phys.org/technology-news/software/)
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.